raiden/app.py

```diff
@@ -84,11 +85,24 @@ def __init__(
             default_registry: TokenNetworkRegistry,
             default_secret_registry: SecretRegistry,
             transport,
+            settle_timeout_min: int,
+            settle_timeout_max: int,
```

Comment:
This is really weird, what is the point of having two additional arguments to validate another argument?